### PR TITLE
[Documentation] Update internal documentation

### PR DESCRIPTION
**Pull request by Batta32**
Related to 3689

### Purpose
*What is the context of this pull request? Why is it being done?*
The [public internal document](https://github.com/microsoft/botframework-solutions) is outdated

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
- Improve documents
- Remove empty documents
- Have parity between botskills' documents
- Fix broken links

The updated files are the followings:
- samples/android/clients/EventCompanion/readme.md
- samples/csharp/assistants/enterprise-assistant/README.md
- sdk/typescript/libraries/bot-solutions/README.md
- templates/csharp/README.md
- templates/csharp/Skill/Skill/readme.md
- templates/csharp/VA/VA/readme.md
- templates/typescript/generator-bot-virtualassistant/README.md
- templates/typescript/generator-bot-virtualassistant/generators/app/README.md
- templates/typescript/generator-bot-virtualassistant/generators/app/templates/sample-assistant/README.md
- templates/typescript/generator-bot-virtualassistant/generators/app/templates/sample-assistant/pipeline/README.md
- templates/typescript/generator-bot-virtualassistant/generators/skill/README.md
- templates/typescript/generator-bot-virtualassistant/generators/skill/templates/sample-skill/README.md
- templates/typescript/generator-bot-virtualassistant/generators/skill/templates/sample-skill/pipeline/README.md
- templates/typescript/samples/sample-assistant/README.md
- templates/typescript/samples/sample-assistant/pipeline/README.md
- templates/typescript/samples/sample-skill/README.md
- templates/typescript/samples/sample-skill/pipeline/README.md
- tools/botskills/README.md
- tools/botskills/docs/commands/connect.md
- tools/botskills/docs/commands/disconnect.md
- tools/botskills/docs/commands/list.md
- tools/botskills/docs/commands/migrate.md
- tools/botskills/docs/commands/refresh.md
- tools/botskills/docs/commands/update.md

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
It was rendered manually in order to check the result of the changes

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
\- 

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [X] I have updated related documentation

### CODE REVIEW

**Review by VictorGrycuk — APPROVED** [review]
LGTM

File: templates/typescript/generator-bot-virtualassistant/generators/app/README.md [hunk 1]
```diff
@@ -1,68 +1,97 @@
...
+Description: A description for my new assistant
+Selected languages: en-us,es-es
+Path: <ASSISTANT_GENERATION_PATH>
 ```
 
-**WARNING:** The process will fail if it finds another folder with the same name of the new assistant.
-
-**NOTE:** Remind to have an **unique** assistant's name for deployment steps. 
-
-**NOTE:** After generating your sample, you can check its README for more information on how to deploy and test it. You can find it in the root folder of your newly created sample or [here](https://microsoft.github.io/botframework-solutions/virtual-assistant/tutorials/create-assistant/typescript/4-provision-you-azure-resources/).
```
Comment (fran893) [inline templates/typescript/generator-bot-virtualassistant/generators/app/README.md:65]: This warning and the notes, should  be removed? the **warning** is an error that actually exists

**Review by FedericoBernal — APPROVED** [review]
LGTM

File: templates/typescript/generator-bot-virtualassistant/generators/app/README.md (hunk 1 above)
Comment (Batta32) [reply templates/typescript/generator-bot-virtualassistant/generators/app/README.md:65]: I agree with you, however:

- The Warning `The process will fail if it finds another folder with the same name of the new assistant` it's redundant as it's explained in the table
- The Note `Remind to have an **unique** assistant's name for deployment steps.` is redundant as the `assistantName` variable contains `Unique name of new assistant (by default takes `sample-assistant`)` description.
- The Note `After generating your sample, you can check its README for more information on how to deploy and test it. You can find it in the root folder of your newly created sample or [here](https://microsoft.github.io/botframework-solutions/virtual-assistant/tutorials/create-assistant/typescript/4-provision-you-azure-resources/).` it's not adding information as this readme should only contains the result of the generator.

**Review by fran893 — APPROVED** [review]

**Review by matiasroldan6 — APPROVED** [review]

**Pull request merged**